pythonwhat/checks/has_funcs.py

```diff
@@ -349,15 +349,19 @@ def has_expr(
         if isinstance(eval_sol, str):
             fmt_kwargs["sol_eval"] = '\'{}\''.format(fmt_kwargs["sol_eval"])
 
+    # reformat student evaluation string if it is too long or contains newlines
+    if incorrect_msg == DEFAULT_INCORRECT_MSG:
```

Comment:
Is this if needed, I can't think of situation in which you ever want the long message. Not 100% sure